src/core_modules/capture-core/components/D2Form/index.js

```diff
@@ -1,4 +1,3 @@
 // @flow
-
 export { asyncHandlerActionTypes, asyncUpdateFieldEpic } from './asyncHandlerHOC';
-
+export { D2Form } from './D2Form.container';
```

Comment:
since I was working on the D2Form to remove the data on unmount - something that in the end we decided not to do - nevertheless I took a moment and exported this to follow our convention with the `index` file